### ISSUE TITLE
** BUILD FAILED **

### ISSUE THREAD

**Issue by Ryanke** [stephencelis/SQLite.swift#528]
-scheme "SQLite iOS" -configuration Release -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean build) failed with exit code 65:
** CLEAN FAILED **

The following build commands failed:
	Check dependencies
(1 failure)

The following build commands failed:
	Check dependencies
(1 failure)

**Comment by tedhenry100** [comment]
I saw this error yesterday too when I tried to use Carthage to install SQLite.swift as a dependency of my project.

I was able to get SQLite.swift to build if I installed manually and used Command-B to initiate the build.

**Comment by Ryanke** [comment]
did it's work, can you tell me you step ?

**Comment by jberkel** [comment]
Can you retry with the latest instructions?

**Comment by tedhenry100** [comment]
I am using Xcode 8.0.

I changed my Cartfile line from what was in the old SQLite.swift readme

    github "stephencelis/SQLite.swift" ~> 0.10.1

to what is now in the new SQLite.swift readme

    github "stephencelis/SQLite.swift" ~> 0.11.0

and then the build initiated by `carthage update` did succeed.

I did see a screenful of warnings at the end of the build that were like

    <module-includes>:1:1: warning: umbrella header for module 'SQLite' does not include header 'sqlite3.h'

After following the Carthage instructions for [Adding frameworks to an application](https://github.com/Carthage/Carthage#adding-frameworks-to-an-application),  I can now `import SQLite` in my `AppDelegate` file and building my project succeeds.
